### DISCUSSION TITLE
Is a picker supposed to just look like text on android?

### DISCUSSION THREAD

**Discussion by Core121** [dotnet/maui#8767] [Q&A]
On android my pickers look like text without any indication to click them to select from a list. Even if I put the picker title as "Select a Category" it looks like a label instead of a clickable item. Is the picker supposed to look like this or am I doing something wrong? I'd rather not have to wrap my picker in a frame every time if it should look clickable out of the box.
![image](https://user-images.githubusercontent.com/42871786/179261106-d41dae45-a47b-498f-ac50-b8272cac2076.png)

**Comment by digitaldirk** [comment]
The [Xamarin Android Picker](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/picker/) had an underline, the MAUI Android picker does not (as of writing this post).  So this is either an potential bug or a design choice. @hartez or @jsuarezruiz can anyone on the team weigh in on this please?

You could also make a style for the picker to make it look more "clickable" or make a custom control that wraps the picker with a frame or something similar so you don't have to keep doing that in your code every time.

**Comment by bryancass** [comment]
I would like the ability to make the Picker field look like other Entries - or at least look 'clickable' - in .Net MAUI.  Can we add TextDecorations to the Picker please?
